src/reports/package/accessibilityInsightsReport.d.ts

```diff
@@ -61,11 +61,18 @@ declare namespace AccessibilityInsightsReport {
         ruleUrl: string
     }
 
+    export type HowToFixData = {
+        any: FormattedCheckResult[],
```

Comment:
`FormattedCheckResult` doesn't seem to be defined or explicitly imported from this file (I think it's coming implicitly from `scanner.d.ts`, so I'm not sure it's going to be usable for a user of the package. It's a pretty short declaration so I'd lean towards just repeating it here.